packages/manager/modules/bm-server-components/src/ipmi/controller.js

```diff
@@ -608,12 +609,21 @@ export default class BmServerComponentsIpmiController {
   }
 
   downloadApplet() {
-    this.trackClick('access-kvm-java');
     this.$window.open(
       `data:application/x-java-jnlp-file,${this.appletToDownload}`,
     );
   }
 
+  goToKvmConsole() {
+    this.trackClick('access-kvm-console');
+    this.$window.open(this.loader.kvmUrl, '_blank', 'noopener');
+  }
+
+  goToSolConsole() {
```

Comment:
**question:** Is this function used? Don't see any reference to it. I guess it's supposed to be event handler for one of the buttons.